.github/workflows/test_examples.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
     name: Examples Pass
     runs-on: ubuntu-latest
     steps:
-      - uses: LouisBrunner/checks-action@v1.1.2
+      - uses: LouisBrunner/checks-action@v1.2.0
```

Comment:
I think you missed a similar update to workflows/pr.yaml line 108.

Reply:
Good catch! Addressed in the latest commit.